### PR TITLE
Duplicate titles checked

### LINKED ISSUE #9: Prevent Duplicate titles
If someone enters a duplicate title, you must notify them. Don't use alerts. Just display the message in red below the text box or somewhere it is pleasant

### PR DESCRIPTION
**Pull request by ananya-mohanty**
__9__

Fixes #9 

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

**Comment by abhishekmsingh** [comment]
@ananya-mohanty ur repo is behind the main repo.by many commits
please perform:
$ git pull upstream master --rebase
$ git push

You may require to set upstream( the main repo )

**Comment by ananya-mohanty** [comment]
Please check now

File: pages/index.html [hunk 1]
```diff
@@ -43,48 +44,44 @@ <h1 class="display-5">Add Items</h1>
...
-
-        <span style="float:left;">
-        <b>   LiBook</b> <i>made by</i> <b >GeekHaven.  </b>
-        </span>
-        <span style="float:right;">
-        <b>Contact:</b> +91 9730000080 , 9000087608
-        </span>
-        <i>© OpenCode Participants</i>
-      </div>
-    </footer>
```
Comment (abhishekmsingh) [inline pages/index.html]: This footer along with some other function will get removed if I merge your pr @ananya-mohanty. Can you please check why?

Comment (ananya-mohanty) [reply pages/index.html]: Altered that. Please check now!

File: scripts/index.js [hunk 2]
```diff
@@ -11,12 +11,33 @@ updateList();
...
+            insertCard(addTxt.value);
+            addTxt.value="";
+            i++;
+        }
+   // localStorage.setItem(i, addTxt.value);
     //Removing 'Your title will appear here' message after first input
-    if(localStorage.length ==1) titles.removeChild(titleMessage);
-    insertCard(addTxt.value);
-    addTxt.value="";
-    i++;
```
Comment (abhishekmsingh) [inline scripts/index.js:19]: This part @ananya-mohanty

Comment (abhishekmsingh) [reply scripts/index.js:19]: oh sorry you did it above

**Comment by abhishekmsingh** [comment]
Last part just give me a live link

**Comment by abhishekmsingh** [comment]
For generating live link,
go to your repository settings, scroll down to github pages section and select master branch in the sources. The url you get just give me that.

**Comment by ananya-mohanty** [comment]
__9__
 Live Link :  https://ananya-mohanty.github.io/libook/

**Pull request merged**